en/translation.json

```diff
@@ -111,7 +111,6 @@
     "moduleStatus": {
         "hour": "hour",
         "hours": "hours",
-        "certificate": "Certificate",
```

Comment:
Looks like this this PR removed this key that is now making the builds on master fail. FYI @joewoodson (it was you that updated the git ref to the translations)

Reply:
But i somehow can't just recreate the key, I get this weird error message from Weblate: `Error in parameter context: Context would be created as "algoliaResults.certificate"` 🤔

Reply:
yeah I noticed this as well. I wanted to update the submodule to merge a PR from Karim, and I didn't realize this translation was deleted. once the [other PR](https://github.com/kiron/campus-frontend/pull/898) is merged then the test should be passing again.

Reply:
Ahh ok, it was deleted on purpose then, cool! Yepp, passing now, thanks.